NSS-Bashful-Turtles/Yak-Social-Network
# C dlogin

Files changed: 
1. index.js
2. LoginRegistration
3. login.js

***
How to test:
```
git fetch --all
git checkout CDlogin
```

1. Type in existing user name and password to respective input fields.  On submit, the session storage should have the "userId" saved as the usernames id.
2. Redo step 1, but check the Remember Me? checkbox.  On submit, the session storage and local storage should have the "userId" saved as the usernames id.
3.  If an incorrect username or password is submitted, an alert should appear.
